src/Microsoft.ML.Data/Utils/LossFunctions.cs

```diff
@@ -455,7 +466,7 @@ public sealed class Options : ISupportRegressionLossFactory
         private readonly Double _index1; // 1 minus the index parameter.
         private readonly Double _index2; // 2 minus the index parameter.
 
-        public TweedieLoss(Options options)
+        private TweedieLoss(Options options)
```

Comment:
>private [](start = 8, length = 7)

why this one is private why every other constructor with options is internal?

Reply:
others cant become internal due to build dependencies. This one can. 

---
In reply to: [264448101](https://github.com/dotnet/machinelearning/pull/2915#discussion_r264448101) [](ancestors = 264448101)